### PR TITLE
Adds endpoints for players/:id GET, PATCH and DELETE

### PR DESCRIPTION
**Pull request by MarchandMD**
# What does this PR do? 
- adds additional endpoints to GET an individual player
- DELETE an individual player
- PATCH an individual player

# How does this PR accomplish that? 
- see the git diffs in files changed

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const cors = require('cors');
 require('dotenv/config');
 
 const app = express();
+
```
Comment (MarchandMD) [inline src/app.js:9]: just some extra whitespace for readability

File: src/app.js [hunk 2]
```diff
@@ -125,6 +126,7 @@ mongoose
 	.then(() => console.log('DB Connected '))
 	.catch((err) => console.log('error'));
 
+// server up
```
Comment (MarchandMD) [inline src/app.js:129]: just a comment for readability

File: src/routes/players.js [hunk 3]
```diff
@@ -57,4 +57,85 @@ playerRouter.post("/", (request, response) => {
         .catch((err) => response.status(400).json("Request Failed"))
 })
 
+/**
```
Comment (MarchandMD) [inline src/routes/players.js:60]: lines 61-82 are the comments to add Swagger documentation at `/apidocs` and the code to get a single player

lines 85-113 are the comments to add Swagger documentation at `/apidocs` and the code to PATCH a single player

lines 116-139 are the comments to add Swagger documentation at `/apidocs` and the code to DELETE a single player

File: src/routes/tasks.js [hunk 4]
```diff
@@ -116,7 +116,7 @@ router.patch("/:id", (request, response) => {
  * /tasks/{taskId}:
  *  delete:
  *    tags:
- *      - [task/:id]
+ *      - [tasks/:id]
```
Comment (MarchandMD) [inline src/routes/tasks.js:119]: turns `task` into `tasks`

**Review by msakr21 — APPROVED** [review]
Looks good!

**Pull request merged**